### ISSUE TITLE
Nix flake broken

### ISSUE THREAD

**Issue by wrocketdev** [Jas-SinghFSU/HyprPanel#153]
I think the fact that the `default.nix` file has been moved to a subfolder is breaking the buildPhase.

```
Running phase: buildPhase
error: ModuleNotFound resolving "./main.ts" (entry point)
```
Can you look into it ?

Thanks (:

**Comment by orxngc** [comment]
![image](https://github.com/user-attachments/assets/ab9912c5-1016-40c0-a076-b2f4dd098f2a)
Interesting

**Comment by Jas-SinghFSU** [comment]
@wrocketdev @orxngc Does this work if you change it to `../main.ts`?

**Comment by orxngc** [comment]
> Does this work if you change it to `../main.ts`?

No.

![image](https://github.com/user-attachments/assets/ad077307-ec23-4d3f-af1a-16e53ebbd328)

**Comment by Jas-SinghFSU** [comment]
Sorry, what if you change the src?
```
  pname = "hyprpanel";
  config = stdenv.mkDerivation {
    inherit pname;
    version = "latest";
  // Changed the source from ./. to ../.
    src = ../.;
```

**Comment by orxngc** [comment]
That does work, actually. #157

**Comment by Jas-SinghFSU** [comment]
Closed by #158
